File: Entidades/Reseña.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entidades
{
	public class Reseña
	{
		[Key]
		public int IDReseña { get; set; }
		public int LicID { get; set; }
        public int JuegoID { get; set; }
        public string? Texto { get; set; }
        public int Numero { get; set; }

	}
}

File: Entidades/Reseña.js
const { Sequelize, DataTypes, Model } = require('sequelize');
const { sequelize } = require('../db.js');

class Reseña extends Model {}

Reseña.init({
    IDReseña: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true
    },
    LicID: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    JuegoID: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    Texto: {
      type: DataTypes.STRING,
      allowNull: true
    },
    Numero: {
      type: DataTypes.INTEGER,
      allowNull: true
    }
}, {

sequelize,
modelName: 'Reseña',
tableName: 'reseña',
timestamps: false
});

module.exports = Reseña;